examples/scripts/my_test.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2018, Terry Jones <terry.jones@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: my_test
+
+short_description: This is my test module
+
+version_added: "2.4"
+
+description: This is my longer description explaining my test module.
+
+options:
+    name:
+        description:
+            - This is the message to send to the test module.
+        required: true
+        type: str
+    new:
+        description:
+            - Control to demo if the result of this module is changed or not.
+        required: false
+        type: bool
+
+extends_documentation_fragment:
+    - azure
```

Comment:
```suggestion
```
it could be a bit confusing for newcomers, i'd avoid such unnecessary things in basic examples